src/drive/web/modules/navigation/Breadcrumb/components/DesktopBreadcrumb/DesktopBreadcrumb.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useMemo, useState } from 'react'
+import classNames from 'classnames'
+import { default as BreadcrumbMui } from '@material-ui/core/Breadcrumbs/Breadcrumbs'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import RightIcon from 'cozy-ui/transpiled/react/Icons/Right'
+
+import styles from 'drive/web/modules/navigation/Breadcrumb/breadcrumb.styl'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import ActionMenu, { ActionMenuItem } from 'cozy-ui/transpiled/react/ActionMenu'
+
+const DesktopBreadcrumb = ({ onBreadcrumbClick, path }) => {
+  const { t } = useI18n()
+
+  const expandText = useMemo(() => t('breadcrumb.label'), [t])
+  const [dropdownTrigger, setDropdownTrigger] = useState(
+    document.querySelector(`[aria-label="${expandText}"]`)
```

Comment:
querySelector can be expensive, right? Should we memoize it ? Or at least store it in order to do not make the call again few lines below? https://github.com/cozy/cozy-drive/pull/2454/files#diff-e47dde8b6893b810a79daecc8827c614ba41ad561395c99789db4027ee222ba2R30

Reply:
good point. That is what was implemented at first. However we faced problems on the public view, when navigating between pages. And we had to come back to this implementation.

So no we don't want to memoize it, because when the second call (line 30) is made, the dropdown trigger could have literally changed from "hidden when less than 3 elements", to "displayed"

The sadness of this code is to have to play with internal component of Material UI... But that was the simplest solution found.